docs/reference/configuration/data_types.rst

```diff
@@ -72,167 +61,352 @@ This section only applies if a fixed seed is used:
+- **<standard_deviation>**: Indicates the standard deviation of the normal distribution.
+
+Example
+~~~~~~~
+For a Gaussian distribution with mean 0 and standard deviation 1 assigned to the variable VAR:
+::
+
+   VAR NORMAL 0 1
+
+.. image:: fig/normal.png
+
+Notes
+~~~~~
+The ``NORMAL`` keyword is integral for scenarios demanding priors that reflect typical real-world data patterns, as the Gaussian distribution is prevalent in many natural phenomena.
 
-LOGNORMAL
-,,,,,,,,,
+LOGNORMAL: Log Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A stochastic variable is log normally distributed if the logarithm of the
-variable is normally distributed. In other words, if X is normally distributed,
-then Y = exp(X) is log normally distributed.
+The ``LOGNORMAL`` keyword is used to assign a log normal prior to a variable. A variable is considered log normally distributed if the logarithm of that variable follows a normal distribution. 
+In mathematical terms, if :math:`X` is normally distributed, then :math:`Y = e^X` is log normally distributed.
 
-A log normal prior is suited to model positive quantities with a heavy tail
-(tendency to take large values). To set a log normal prior, use the keyword
-LOGNORMAL. It takes two arguments, the mean and standard deviation of the
-*logarithm* of the variable:
+Log normal priors are especially suitable for modeling positive quantities that exhibit a heavy tail, indicating a tendency for the quantity to occasionally take large values.
 
+Syntax
+~~~~~~
 ::
 
-   VAR2   LOGNORMAL  0  1
+  VAR LOGNORMAL <log_mean> <log_standard_deviation>
 
-TRUNCATED_NORMAL
-,,,,,,,,,,,,,,,,,
+Parameters
+~~~~~~~~~~
+- **<log_mean>**: Represents the mean of the logarithm of the variable.
+- **<log_standard_deviation>**: Denotes the standard deviation of the logarithm of the variable.
+
+Example
+~~~~~~~
+For a variable with its logarithm having a mean of 0 and a standard deviation of 1:
+::
+
+   VAR LOGNORMAL 0 1
 
-This *TRUNCATED_NORMAL* distribution works as follows:
+.. image:: fig/lognormal.png
 
-   1. Draw random variable :math:`X \sim N(\mu,\sigma)`
-   2. Clamp X to the interval [min, max]
+TRUNCATED_NORMAL: Truncated Normal Distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This is **not** a proper normal distribution; hence the
-clamping to `[min,max]` should be an exceptional event. To configure
-this distribution for a situation with mean 1, standard deviation 0.25
-and hard limits 0 and 10:
+The ``TRUNCATED_NORMAL`` keyword is utilized to assign a truncated normal distribution to a variable.
+This distribution works as follows:
 
+   1. Draw a random variable :math:`X \sim N(\mu,\sigma)`.
+   2. Clamp :math:`X` to the interval [min, max].
+
+It's important to note that this is **not** a proper normal distribution due to the constraints applied.
+Thus, the act of clamping values to the `[min,max]` interval should be an exceptional event. 
```

Comment:
Most of the distributions we expose are not normal distributions. Should it be an exceptional event to use non-normal distributions, or why/how is this different?

Reply:
I think this is talking about the `TRUNCATED_NORMAL` (or clamped normal) specifically but not sure.
Perhaps it is because you can end up with too high probabilities of end-points because of the clamping?